image/image.go

```diff
@@ -49,6 +49,13 @@ var validRefMediaTypes = []string{
 }
 
 func validate(w walker, refs []string, out *log.Logger) error {
+	var tmperr error
```

Comment:
No need for `tmperr`.  I'd rather have:

```go
func validate(w walker, refs []string, out *log.Logger) (err error) {
```

above, use `err` for `refsNameRegexpCheck`, and possibly shift an `:=` → `=` below where we currently setup `err`.